lib/view/pages/stats/stats_games.dart

```diff
@@ -0,0 +1,127 @@
+import 'package:flutter/material.dart';
+import 'package:gameonconnect/services/stats_S/stats_games_service.dart';
+import 'package:timeago/timeago.dart' as timeago;
+// import 'package:logger/logger.dart'; // Add this line for logging
+
+// final logger = Logger(); // Initialize the logger
+
```

Comment:
out of interests sake, what did you use the logger for?

Reply:
I used it to figure out why my code wasn't working. So I followed the code and added logging until I found the problem.

Reply:
is it like print? What is different between print and logger?

Reply:
It's similar to print. The output in the terminal is just highlighted and formatted so that its easier to see what's happening.

Reply:
ah okay interesting, thank you